docs/tf/ray_usage.rst

```diff
@@ -0,0 +1,260 @@
+**************************************************
+Ray Usage Example for Node Classification with GAT
+**************************************************
+
+.. code-block:: python
```

Comment:
Please add some comments to the code. It is not self-explanatory for someone who starts directly with deepgnn and TF.

Reply:
This is a copy of the node class example, only with small changes to make it use Ray.

Reply:
A person who works with TF doesn't know about a pytorch copy and that they should start with it. Also why should they?

Reply:
That should be for a different PR then, this is meant to be a new version of the tf_node_class that uses Ray. The original version has no documentation either,